src/game/client/water/w3dwater.cpp

```diff
@@ -1087,17 +1087,16 @@ void WaterRenderObjClass::Draw_Sea(RenderInfoClass &rinfo)
         ww3d.m[2][1] = 1.0f;
         ww3d.m[1][2] = 1.0f;
         ww3d.m[3][3] = 1.0f;
-        Matrix3D m4(m_transform);

```

Comment:
I think this needs checking.

Reply:
Looks safe to do. `m_transform` already is `Matrix3D`, so this just creates a useless copy here and `DX8Wrapper::Set_Transform` just reads it.